Automattic/wp-calypso
# Tiled gallery block: remove un-used code from layout calculations

#### Changes proposed in this Pull Request

Remove some unused code from Tiled gallery block.

#### Testing instructions

- Spin up site either via gutenpack-jn or https://calypso.live/block-editor?branch=update/tiled-gallery-remove-dead-code
- Add tiled gallery block to post
- Confirm everything works with rectangular layout
